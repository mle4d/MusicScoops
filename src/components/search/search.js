import React from 'react';
import PropTypes from 'prop-types';


function Search({ artist, inputChange, onClick }) {
  return (
    <div>
      <input type='text' name='artist' value={artist} onChange={inputChange} />
      <button onClick={onClick}>Search</button>
    </div>
  );
}

Search.propTypes = {
  artist: PropTypes.string.isRequired,
  inputChange: PropTypes.func.isRequired,
  onClick: PropTypes.func.isRequired
};

export default Search;
