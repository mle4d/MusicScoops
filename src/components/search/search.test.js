import React from 'react';
import { shallow } from 'enzyme';
import search from './search';

describe('search component', () => {
  it('renders search', () => {
    const wrapper = shallow(<search />);
    expect(wrapper).toMatchSnapshot();
  });
});
