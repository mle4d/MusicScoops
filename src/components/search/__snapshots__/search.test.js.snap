// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search component renders search 1`] = `<search />`;
